dom-querying/main.js

```diff
@@ -0,0 +1,17 @@
+const heading = document.querySelector('h1');
+console.log('$heading', heading);
+
+console.dir('$heading', heading);
+
+const explanation = document.querySelector('explanation');
```

Comment:
This should have a `#` inside the `querySelector`. You're querying for an `id` of `explanation`